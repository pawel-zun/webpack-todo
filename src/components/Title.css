.Title {
	text-align: center;
}

.Title__headline {

}

.Title__tasks {

}

.Title__tasks_number {
	font-size: 24px;
	font-weight: 700;
	color: #F4511E;
}