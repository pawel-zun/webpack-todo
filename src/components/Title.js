import React from 'react';
import style from './Title.css';

class Title extends React.Component {
  render() {
    return (
    	<div className={style.Title}>
	    	<h1 className={style.Title__headline}>{this.props.headline}</h1>
	    	<p className={style.Title__tasks}>There are: <span className={style.Title__tasks_number}>{this.props.howMany}</span> tasks to complete.</p>
    	</div>
    );
  }
}

export default Title;