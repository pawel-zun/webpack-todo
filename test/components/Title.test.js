import React from 'react'
import style from '../../src/components/Title.css';
import Title from '../../src/components/Title'

test('render a title', () => {
  const wrapper = shallow(
      <Title 
        headline="What are you going ToDo?"
        howMany={4} />
  );
  expect(wrapper).toMatchSnapshot();
});
