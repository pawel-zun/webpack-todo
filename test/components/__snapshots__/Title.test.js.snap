// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a title 1`] = `
<div
  className="Title"
>
  <h1
    className="Title__headline"
  >
    What are you going ToDo?
  </h1>
  <p
    className="Title__tasks"
  >
    There are: 
    <span
      className="Title__tasks_number"
    >
      4
    </span>
     tasks to complete.
  </p>
</div>
`;
